src/pilot/deploy-overcloud.py

```diff
@@ -637,6 +278,10 @@ def main():
                             dest="hugepages_size",
                             required=False,
                             help="HugePages size")
+        parser.add_argument("--hostos_cpu_count",
```

Comment:
There might be a check here. What if the user enable numa without specifying any values for hostos_cpu_count? What is expected to happen then?

Reply:
I don't think so this will happen
deployer.py will catch those empty fields in JSON validation
Tried already got following : 
Your setting for Dell NFV Settings::numa_hostos_cpu_count is not valid for the profile csp.It should be set to one of the following options : ["2", "4", "6", "8"]
Your setting for Dell NFV Settings::hpg_size is not valid for the profile csp.It should be set to one of the following options : ["2MB", "1GB"]

Let me know if I am missing something here.

Reply:
What about a manual deployment ?

Reply:
Fixed
Added default values.